earth_enterprise/src/fusion/dbmanifest/dbmanifest.cpp

```diff
@@ -962,18 +963,40 @@ void DbManifest::GetMapsJsonFiles(
   LoadLocales(&locale_set);
   // Need to parse the stream url.
   QUrl qurl(stream_url.c_str());
+  int version;
   for (int i = -1; i < static_cast<int>(locale_set.supportedLocales.size());
        ++i) {
     std::string locale = i < 0 ?
         kDefaultLocaleSuffix : std::string(locale_set.supportedLocales[i]);
 
+    // Find the version parameter.
+    std::string versionParam = "v=";
+    std::string query = qurl.query();
+    int vIndex = query.find(versionParam) + 3;
```

Comment:
I think you should check for the fact find might not find the string `v=` before you add anything to it.  If find returns `std::string::npos` that means the call did not include the version parameter and we should use the default (1 in the first release we have this change and 2 in the next release after).

Also, should you not be adding `2` instead of `3`?